Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -68,6 +69,12 @@ public class SeleniumDriver : DriverBase, IVirtualDriver, IWindowExplorer, IVisu
     {
         protected IDevToolsSession Session;
         DevToolsSession devToolsSession;
+        Dictionary<string, object> networklogdict;
```

Comment:
change name to networkResponseLogDict and networkRequestLogDict

Reply:
Changes Done